psat_server_web/atlas/templates/create_user.html

```diff
@@ -0,0 +1,165 @@
+{% extends "base_bootstrap.html" %}
+
+{% block title %}Create New User - ATLAS{% endblock %}
+
+{% block content %}
+<div class="container mt-4">
+    <div class="row">
+        <div class="col-md-8 offset-md-2">
+            <div class="card">
+                <div class="card-header bg-primary text-white">
+                    <h1 class="mb-0"><i class="fa-solid fa-user-plus"></i> Create New User</h1>
+                </div>
+                <div class="card-body">
+                    {% if messages %}
+                        {% for message in messages %}
+                            <div class="alert alert-{{ message.tags }} alert-dismissible fade show" role="alert">
+                                {{ message }}
+                                <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+                                    <span aria-hidden="true">&times;</span>
+                                </button>
+                            </div>
+                        {% endfor %}
+                    {% endif %}
+
+                    <form method="post">
+                        {% csrf_token %}
+                        
+                        <div class="row">
+                            <div class="col-md-6">
+                                <div class="form-group">
+                                    <label for="{{ form.username.id_for_label }}">Username *</label>
+                                    {{ form.username }}
+                                    {% if form.username.errors %}
+                                        <div class="text-danger">{{ form.username.errors }}</div>
+                                    {% endif %}
+                                    <small class="form-text text-muted">{{ form.username.help_text }}</small>
```

Comment:
Form hints should be linked to the text input, using either `aria-labelledby` or `aria-describedby`.

Reply:
This is very useful for adding accessible descriptions to Django's form fields.
https://www.valentinog.com/blog/django-accessible-web-forms/